### ISSUE TITLE
Document new doOnEmpty() operator

### ISSUE THREAD

**Issue by DavidMGross** [ReactiveX/reactivex.github.io#183]
See ReactiveX/RxJava#3624 and ReactiveX/RxJava#3621

**Comment by thomasnield** [comment]
I'm curious, are we talking about more than just Javadocs? Does this entail marble diagrams?

**Comment by DavidMGross** [comment]
Yeah; this is about the reactivex website, so marble diagrams and an entry
on the web page.

On Sun, Jan 24, 2016 at 12:00 AM, Thomas Nield <notifications@github.com>
wrote:

> I'm curious, are we talking about more than just Javadocs? Does this
> entail marble diagrams?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ReactiveX/reactivex.github.io/issues/183#issuecomment-174265848>
> .
>



-- 
David M. Gross
PLP Consulting

**Comment by akarnokd** [comment]
Can be closed, we decided this operator won't be part of RxJava as for now.
